Code/Framework/AzCore/AzCore/Task/TaskGraph.h

```diff
@@ -99,6 +102,8 @@ namespace AZ
     class TaskGraph final
     {
     public:
+        // ! The supplied string label is expected to be a string literal or otherwise outlive the lifetime of this TG.
+        explicit TaskGraph(const char* graphlabel);
```

Comment:
Can you also delete a constructor that accepts a nullptr to prevent the user for trying to construct with a ` nullptr` such as
```
TaskGraph tg(nullptr);
```
 ```suggestion 
         explicit TaskGraph(const char* graphlabel);
         TaskGraph(AZStd::nullptr_t) = delete;
```